spec/requests/users_spec.rb

```diff
@@ -3,60 +3,62 @@
 require 'rails_helper'
 
 RSpec.describe 'users', type: :request do
+  let(:user) { create(:user) }
+
+  context 'when not logged in' do
+    describe '/users/sign_in' do
+      it 'returns http 200' do
+        post user_session_path, params: {email: user.email, password: user.password}
+        expect(response).to have_http_status(:ok)
+      end
+    end
+  end
+
   context 'when logged in' do
-    let(:user) { create(:user) }
     let(:user_params) { attributes_for(:user) }
```

Comment:
```suggestion
    let(:user_params) { attributes_for(:user, current_password: user.password) }
```

Reply:
It also works without here...

Reply:
Are you sure about that? I think, it only works because you check for status 200. That will, however, include a validation error as the current_password is required to update the user (check the `response.body` during the test run). If an error is included (e.g. the password is missing), no redirect will be returned but the same page with a validation error (status 200). If the user was updated, he / she gets redirected.

Reply:
Good point, bad test. 😜 I'll check how I can improve this.